CriteoPublisherSdk/Sources/Util/CR_URLResolver.m

```diff
@@ -24,11 +24,12 @@
 
 @interface CR_URLResolution ()
 @property(nonatomic, strong) NSURL *URL;
-+ (instancetype)resolutionWithType:(CR_URLResolutionType)type;
++ (instancetype)resolutionWithType:(CR_URLResolutionType)type URL:(NSURL *)URL;
 + (instancetype)resolutionError:(NSError *)error;
 @end
 
 @interface CR_URLResolver () <NSURLSessionDataDelegate>
+@property(atomic) BOOL resolved;
```

Comment:
Could you explain in the commit message why you want the resolutionHandler to be called only once ?
I guess that there might be a good reason. If so, what do you think about showing it with a test ?

Reply:
Done